script/deploy/healthCheck.mts

```diff
@@ -33,7 +32,6 @@ const corePeriphery = [
   'Receiver',
```

Comment:
I am wondering if it makes sense to read these from the config.sh (the config for our scripts) cause there we maintain this info already and it doesnt make so much sense to maintain it in different places. 
Alternatively we could consider to have subfolders "core" in our facets and periphery folder. Then we dont need to maintain the info anywhere but could just extract all names from the folder. What do you think?

Reply:
I agree with having a separate folder for core.